bin/server.rs

```diff
@@ -153,10 +153,12 @@ async fn main() -> anyhow::Result<()> {
     // All other cases are considered errors worthy of failing.
     let keyring: KeyRing = match std::fs::metadata(&keyring_file) {
         Ok(md) if md.is_file() => load_toml(keyring_file).await?,
-        Ok(_) => Err(anyhow::anyhow!(
-            "Expected {} to be a regular file",
-            keyring_file.display()
-        ))?,
+        Ok(_) => {
+            return Err(anyhow::anyhow!(
```

Comment:
Can this be reduced to `anyhow::bail!`? I always forget to use `bail!()`.

Reply:
Oh yeah I forgot about bail. Will fix